db/db.js

```diff
@@ -0,0 +1,115 @@
+const Sequelize = require('sequelize');
+const sequelize = new Sequelize('postgres://xuvncdsf:5tB-jsNR5ko5TI4l0OKuRFuodEMAtmQE@rogue.db.elephantsql.com:5432/xuvncdsf') 
```

Comment:
<important> db connection string should not be on git, it would be best to put it in env variable